src/include/type/type_id.h

```diff
@@ -30,7 +30,8 @@ enum class TypeId : uint8_t {
   DATE,
   VARCHAR,
   VARBINARY,
-  PARAMETER_OFFSET
+  PARAMETER_OFFSET,
+  VARIADIC,
```

Comment:
This isn't a real `Type`, but neither is `PARAMETER_OFFSET`. Will need to refactor later.

Reply:
So the actual type from postgres is "any" (https://github.com/cmu-db/noisepage/issues/1176). We can just rename this to that?

Reply:
I'd say leave as it. This'll need to be refactored later outside this PR.